lifecycle.Rmd

```diff
@@ -0,0 +1,238 @@
+   lab" to which people may submit and discuss ideas, with the system
+   explicitly designed to connect ideas to potential mentors who may guide
+   development towards full packages.
+
+### The Debian System
+
+The development of software for the open-source [Debian Operating
+System](https://debian.org) is guided by Debian Developers and Debian
+Maintainers. Expressed roughly, maintainers are individuals responsible for the
+maintenance of particular pieces of software, while developers engage with
+activities supporting the development of the operating system as a whole. The
+submission and review process for Debian is almost entirely automated, based on
+tools such as their own software checker,
+[`lintian`](https://lintian.debian.org). Debian differs fundamentally from the
+system proposed here in being centred around the trust and verification of
+people rather than software. Submission of software to Debian is largely
+automatic, and bug-free software may often progress automatically through
+various stages towards acceptance. Software may, however, only be submitted by
+official Debian Maintainers or Developers. People can only become developers or
+maintainers through being sponsored by existing members, and are then subject
+to review of the potential contribution they may be able to make to the broader
+Debian community. (Details can be seen in [this chapter of the Debian
+handbook](https://debian-handbook.info/browse/stable/sect.becoming-package-maintainer.html).)
+
+While the general process for software submission and acceptance in Debian may
+not be of direct relevance, their versioning policy may provide a useful mode.
+The ongoing development of both the Debian system and all associated packages
+proceeds in accordance with a versioned [policy
+manual](https://www.debian.org/doc/debian-policy/index.html). All new packages
+must comply to the current standards at the time of submission, and are
+labelled with the latest version of the standards to which they comply, [noting
+that](https://www.debian.org/doc/debian-policy/ch-source.html#standards-conformance),
+
+> For a package to have an old Standards-Version value is not itself a bug ...
+It just means that no-one has yet reviewed the package with changes to
+the standards in mind.
+
+Each new version of the standards is accompanied by a simple
+[checklist](https://www.debian.org/doc/debian-policy/upgrading-checklist.html)
+of differences, explicitly indicating differences with and divergences from
+previous versions. As long as software continues to pass all tests, upgrading
+to current standards remains optional. Failing tests in response to any
+upgrading of standards serve as a trigger for review of software. The nominated
+standards version may only be updated once review has confirmed compliance with
+current standards. We propose to adapt some of these aspects of the Debian
+system in the present project, as described below.
+
+### Other Potential Models
```

Comment:
Another model I was thinking of was internal reviews within a team, like what they do in the tidyverse team and https://reside-ic.github.io/ -- might be off-topic

Reply:
That is very much on-topic, and something we have discussed and considered. Some kind of vision of commitment to involvement beyond singular review, so that potentially multiple people can re-engage with code in semi-formal ways during ongoing development. Definitely aiming for that, but can't really do much in concrete ways until we know more about the shape the whole system might take.